### PR TITLE
added priting of talisman commands when no flags are provided by the user

### PR DESCRIPTION
**Pull request by mabaritw**
Prints usage of Talisman when no flags are provided by the user:
       talisman -scan
       talisman -v

### CODE REVIEW

File: talisman.go [hunk 1]
```diff
@@ -58,6 +58,13 @@ func main() {
 		os.Exit(0)
 	}
 
+	if flag.NFlag() == 0 {
+		fmt.Println(
+			"Usage \"talisman [option]\"\nOptions:\n\t\"-scan\" -> scan git history\n\t\"-v\" -> check Talisman version",
+		)
```
Comment (svishwanath-tw) [inline talisman.go]: Please consider using flag.PrintDefaults, at least.
Stretch goals: switch out flag with one of spf13's packages [pflag](https://github.com/spf13/pflag), [cobra](https://github.com/spf13/cobra) or [viper](https://github.com/spf13/viper)

File: README.md [hunk 2]
```diff
@@ -273,7 +273,7 @@ Talisman also scans the content present in the git history of the repository, th
 
 To run the scanner please "cd" into the directory to be scanned and run the following command
 
-* `talisman scan`
+* `talisman -scan`
```
Comment (svishwanath-tw) [inline README.md]: This looks like an unrelated change

File: talisman.go (hunk 1 above)
Comment (svishwanath-tw) [inline talisman.go]: Please consider using flag.PrintDefaults, at least.
Stretch goals: switch out flag with one of spf13's packages [pflag](https://github.com/spf13/pflag), [cobra](https://github.com/spf13/cobra) or [viper](https://github.com/spf13/viper)

Comment (mabaritw) [reply talisman.go]: Okay, Suhas.
